src/test/java/redis/clients/jedis/doc/SetGetExample.java

```diff
@@ -0,0 +1,35 @@
+//EXAMPLE: set_and_get
+//HIDE_START
+package redis.clients.jedis.doc;
+
+import redis.clients.jedis.Jedis;
+//REMOVE_START
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+//REMOVE_END
+
+public class SetGetExample {
+
+    @Test
+    public void run() {
+
+        Jedis jedis = new Jedis("localhost", 6379);
```

Comment:
Shouldn't we encourage users to use UnifiedJedis and variants?

Reply:
Good question. The basic examples here (e.g., the README in this repo) use Jedis instead of UnifiedJedis. It would be clearly good to use the UnifiedJedis class for any Redis Stack related example. Let me check with Igor.

Reply:
I vote for using UnifiedJedis to make it consistent.